### DISCUSSION TITLE
Documentation improvements

### DISCUSSION THREAD

**Discussion by kamilkisiela** [dotansimha/graphql-code-generator#4312] [General]
A list of things we could improve in documentation and READMEs:
- present a workflow in each plugin (an example of how to actually use the plugin).
- add `documents` in operation-related plugins

@dotansimha @ardatan @Urigo We can track issues with documentation here

**Comment by Manubi** [comment]
Would love to see that as I find the documentation rather useless. At least for Vue.

**Comment by flintforge** [comment]
Shouldn't each plugin provide it's own readme in its source folder ?
Plugins docs are in /[website](https://github.com/dotansimha/graphql-code-generator/tree/master/website)/[src](https://github.com/dotansimha/graphql-code-generator/tree/master/website/src)/[pages](https://github.com/dotansimha/graphql-code-generator/tree/master/website/src/pages)/[plugins](https://github.com/dotansimha/graphql-code-generator/tree/master/website/src/pages/plugins)/

The /website folder could be moved away to it's own repos.

Better get the source & the doc,
rather that get source & (where is the doc?) & go back on the website & search the plugin again & read it online.

**Reply by saihaj** [reply]
> The /website folder could be moved away to it's own repos.

It is easier to manage docs and code in one repo. That way if we add/remove things from code we can reflect in docs.


> Shouldn't each plugin provide it's own readme in its source folder ?

Yes!
